exo-deploy/src/deploy.ls

```diff
@@ -3,6 +3,10 @@ require! {
   'require-yaml'
 }
 
+command-flag = process.argv[2]
```

Comment:
The CLI class should parse all command-line arguments and then call this class with the right parameters.